src/out/admin/js/api-version-selector.js

```diff
@@ -0,0 +1,13 @@
+function selectMethod(selector)
```

Comment:
i think better not to use jquery and rely on vanilla js


    const changeDisabled = function (value) {
        document.querySelectorAll('[name^="confstrs[ffAuth"]').forEach(function (element) {
            element.disabled = value;
        });
    };

    if (selectedVersion === excludedVersion) {
        changeDisabled(true);
    } else {
        changeDisabled(false);
    }
parhaps this could be used?